bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx

```diff
@@ -29,14 +29,16 @@ export default class DropdownListItem extends Component {
 
   render() {
     const { label, description, children, injectRef, tabIndex, onClick, onKeyDown,
-      className, style, separator, intl, } = this.props;
+      className, style, separator, intl, placeInTabOrder, } = this.props;
+
+    let index = (placeInTabOrder) ? 0 : -1;
```

Comment:
Can't you just pass the `tabIndex` that you desire instead of the `placeInTabOrder` prop? What happens if I want a item with a specific `tabIndex`?

Missing PropTypes and DefaultProps for the new property.